test/powershell/engine/COM/COM.Basic.Tests.ps1

```diff
@@ -0,0 +1,48 @@
+
+try {
+    $defaultParamValues = $PSdefaultParameterValues.Clone()
+    $PSDefaultParameterValues["it:skip"] = ![System.Management.Automation.Platform]::IsWindowsDesktop
+
+    Describe 'Basic COM Tests' -Tags "CI" {
```

Comment:
the following might be a more direct test:
```
Describe a {
        BeforeAll {
                $null = new-item -type file $TESTDRIVE/file1
                $null = new-item -type file $TESTDRIVE/file2
                $null = new-item -type file $TESTDRIVE/file3
                $null = new-item -type file $TESTDRIVE/file4
        }
        it b {
                $shell = New-Object -com shell.application
                $folder = $shell.Namespace("$TESTDRIVE")
                $folder.Items().Count | should be 4
                $names = $folder.Items() | %{$_.name}
                $names -join "," | should be "file1,file2,file3,file4"
        }
}
```
you have complete control over what you expect and can validate it